### PR TITLE
Last set of TestPlayer migrations.

### PR DESCRIPTION
**Pull request by kberg**
Please feel free to opine about the use of TestPlayers in Players.spec.ts

### CODE REVIEW

File: tests/Game.spec.ts [hunk 1]
```diff
@@ -393,11 +391,11 @@ describe('Game', function() {
   });
 
   it('Removes Hellas bonus ocean space if player cannot pay', function() {
+    const player = TestPlayers.BLUE.newPlayer();
```
Comment (kberg) [inline tests/Game.spec.ts:394]: I moved this up above the comment as the comment really relates to `secondPlayer`. This repeats for the test method below.

File: tests/Player.spec.ts [hunk 2]
```diff
@@ -1,7 +1,5 @@
 
 import {expect} from 'chai';
-import {Color} from '../src/Color';
-import {Player} from '../src/Player';
```
Comment (kberg) [inline tests/Player.spec.ts:4]: I'll say this: there's nothing at all about the player constructor that is interesting, other than generating the player ID, and at this point I wouldn't bother writing a test for it just so Player was imported. Thoughts? Do you want one of those sanity tests where you call the constructor just to verify its values?

**Comment by kberg** [comment]
Whoops! Will fix.

**Review by nwai90 — APPROVED** [review]

File: tests/Player.spec.ts (hunk 2 above)
Comment (Lynesth) [reply tests/Player.spec.ts:4]: I think one test in "real" conditions would be nice yes. Can be done after you moved the id generation outside of the constructor though.

Comment (kberg) [reply tests/Player.spec.ts:4]: OK. Fascinating that the change I made to support this is actually breaking tests in surprising places. Thankfully they're database tests. (THANK YOU @bafolts)

**Pull request merged**